### DISCUSSION TITLE
Project Status

### DISCUSSION THREAD

**Discussion by nartc** [nartc/mapper#535] [General]
Hi all, 

First of all, thank you for the support to the project thus far, I really really appreciate it, truly.

Second, sorry for neglecting the project in the past several months (closed to a year now I think 🫠).

I have been a bit busy at work and and occupied with some family stuffs. In addition, I've been more involved on the Angular side of things (some of you might know me as an Angular Google Developer Experts) so less time to focus on AutoMapper.

That said, I'm going to transition to a new team at work with focus to build our in-house product rather than consulting so I'll have better time management to jump around my open source projects, AutoMapper is one of those that I will prioritize starting in about 2-3 weeks.

First step is to go through all the outstanding issues and address them one by one. Next, as the TypeScript ecosystem moves forward, I'm starting to think about a schema-based mapper (eg: with `zod`), more on this in the future discussions.

Once again, thank you!

- Chau -

**Comment by abouroubi** [comment]
Hello @nartc, hope everything is fine.
Any news on getting back on the project ? Or maybe giving access to someone else to merge PRs ?

**Comment by HppZ** [comment]
is this repo still active?
